### PR TITLE
feat!: added greater support for dependancy injection through tsyringe

### PR DESCRIPTION
**Pull request by wadeking98**
replaced our old useConfiguration hook for the updated tsyringe container

### CODE REVIEW

File: app/container-imp.ts [hunk 1]
```diff
@@ -107,6 +151,92 @@ export class AppContainer implements Container {
...
+              accessibilityLabel: this.t('Settings.PlayWithBCWallet'),
+              testID: testIdWithKey('PlayWithBCWallet'),
+              onPress: () => Linking.openURL('https://digital.gov.bc.ca/digital-trust/showcase/'),
+            },
+          ],
+        },
+      ],
+      enableTours: false,
+      supportedLanguages: ['en'],
+      showPreface: false,
```
Comment (bryce-mcmath) [inline app/container-imp.ts]: Shouldn't `showPreface` be true?

File: app/container-imp.ts [hunk 2]
```diff
@@ -107,6 +151,92 @@ export class AppContainer implements Container {
...
+            {
+              title: this.t('Settings.PlayWithBCWallet'),
+              accessibilityLabel: this.t('Settings.PlayWithBCWallet'),
+              testID: testIdWithKey('PlayWithBCWallet'),
+              onPress: () => Linking.openURL('https://digital.gov.bc.ca/digital-trust/showcase/'),
+            },
+          ],
+        },
+      ],
+      enableTours: false,
```
Comment (bryce-mcmath) [inline app/container-imp.ts]: Shouldn't `enableTours` be true?

File: app/container-imp.ts [hunk 3]
```diff
@@ -107,6 +151,92 @@ export class AppContainer implements Container {
...
+      supportedLanguages: ['en'],
+      showPreface: false,
+      disableOnboardingSkip: false,
```
Comment (bryce-mcmath) [inline app/container-imp.ts]: Shouldn't `disableOnboardingSkip` be true?

File: app/App.tsx [hunk 4]
```diff
@@ -1,9 +1,5 @@
 import {
-  Stacks,
-  Screens,
   AgentProvider,
-  AnimatedComponentsProvider,
-  animatedComponents,
```
Comment (bryce-mcmath) [inline app/App.tsx:6]: Why were these two removed? I still see `useAnimatedComponents` throughout Bifold

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
Some small things and a question

File: app/App.tsx (hunk 4 above)
Comment (wadeking98) [reply app/App.tsx:6]: not sure, must have gotten deleted when I was moving the settings to container-imp

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**